torch/_inductor/ir.py

```diff
@@ -3542,7 +3542,8 @@ def apply_constraint(self):
         pass
 
     def codegen_const_args(self):
-        return map(V.graph.wrapper_code.val_to_arg_str, self.constant_args)
+        # FIXME: separate invocation for cpp arg strs?
```

Comment:
maybe this is fine since looking at its caller, `codegen_args`, it looks like the codegen is meant to target Python only

I'm overall confused by the difference between ExternKernels and FallbackKernels though, and how codegen logic is split between the two. Both of them define `codegen_args`, but they share a single implementation of `codegen_kwargs`...